README.md

```diff
@@ -18,12 +18,27 @@ In the main project directory there is a docker-compose.yml file that will build
 
 Before composing the application you must ensure that Docker is installed and functioning. A quick check is to run `docker version` on the command line to see if you get any output. If you get an error, see the Docker set up for your platfrom.
 
- - Ubuntu - this should be as simple as `sudo apt install docker docker-compose docker.io python3-docker python3-dockerpty` 
+ - Ubuntu - this should be as simple as `sudo apt install docker docker-compose docker.io python3-docker python3-dockerpty` . To enable Docker access for the current user, run the following command in the terminal: `sudo usermod -a -G docker $USER`. It's important to note that after running the command, you need to log out and log back in for the changes to take effect.
  - MacOS [Docker Desktop for Mac](https://docs.docker.com/docker-for-mac/install/)
  - Windows 10 [Docker Desktop for Windows 10](https://docs.docker.com/docker-for-windows/install/)
 
 It is recomended to perform the install and "getting started" steps for Docker Desktop before proceeding.
 
+## Important Notice
+
+**Please Read Before Installing Docker Desktop and KVM on Linux**
+
+If you are using a Linux-based operating system, such as Ubuntu, it is important to note that Docker Desktop and KVM (Kernel-based Virtual Machine) are already included in the Linux kernel and installing them separately can lead to conflicts and issues with your environment.
+
+**DO NOT INSTALL Docker Desktop or KVM ON LINUX**
```

Comment:
I would recommend just a little change in the wording to indicate that what you mean is not to install them from third party sources.  We could also add instructions for other Linux distros if that might avoid the issues you saw. I use Fedora with podman, for example.

Reply:
Also, just in terms of process, this should have a related Issue and it should be merged to the `v2` branch first.